content/news/2023-05-14-summer-of-haskell-2023-project-selections.md

```diff
@@ -0,0 +1,150 @@
+---
+title: 'Summer of Haskell 2023 Project Selections'
+author: 'Aaron Allen'
+---
+
+Summer of Haskell 2023 is off to a great start and we have some exciting news
+to share! We received many quality proposals from which to choose and our
+mentors have made their selections. Thanks to our incredibly generous sponsors,
+we have secured funding for 9 projects! Without further ado, here are the
```

Comment:
```suggestion
we have secured funding for nine projects! Without further ado, here are the
```
Rule of thumb for semi-formal writing that I learned in school is to spell out numbers less than 10. If this is too old-fashioned for you, please disregared.